src/common/selftest/selftest_firstlayer_interface.hpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file selftest_firstlayer_interface.hpp
+ * @brief Set of functions adding functionality to selftest classes
+ * Could be solved with multiple inheritance (did not want that)
+ * or Interface (C++ does not have)
```

Comment:
```
@brief Set of functions adding functionality to selftest classes
```
Information value of this statement is low. I can see that it is declared in selftest namespace anyway. There is no class called `selftest`, do you mean `CSelftest`?
```
Could be solved with multiple inheritance (did not want that)
 * or Interface (C++ does not have)
```
1. I don't understand that.
2. If I would understand that still doesn't sound very important to me.
3. Much more useful would be if there was explanation how to use or what is

Reply:
There should be blank space between
` * @brief Set of functions adding functionality to selftest classes`
and
` * Could be solved with multiple inheritance (did not want that)`
otherwise brief description would contain all the text.

Reply:
selftest is not meant as name of class
ok will rewrite it